src/dash-table/derived/cell/inputs.tsx

```diff
@@ -25,15 +26,16 @@ const getter = (
     activeCell: ActiveCell,
     columns: VisibleColumns,
     data: Data,
+    offset: IViewportOffset,
     editable: boolean,
     isFocused: boolean,
     tableId: string,
     dropdowns: any[][],
-    propsFn: () => ICellFactoryOptions
+    propsFn: () => ICellFactoryProps
 ): JSX.Element[][] => mapData(
     (datum, rowIndex) => mapRow(
         (column, columnIndex) => {
-            const active = isActiveCell(activeCell, rowIndex, columnIndex);
+            const active = isActiveCell(activeCell, rowIndex + offset.rows, columnIndex + offset.columns);
```

Comment:
Now need the offset to reference the right row and column (although this one is always 0 for now)